virtcontainers/factory/factory.go

```diff
@@ -22,28 +22,39 @@ type FactoryConfig struct {
 	Template bool
 	Cache    uint
 
-	vmConfig vc.VMConfig
+	VMConfig vc.VMConfig
 }
 
 func (f *FactoryConfig) validate() error {
-	return f.vmConfig.HypervisorConfig.Valid()
+	return f.VMConfig.HypervisorConfig.Valid()
 }
 
 type factory struct {
 	base base.FactoryBase
 }
 
-func NewFactory(config FactoryConfig) (vc.Factory, error) {
+func NewFactory(config FactoryConfig, fetchOnly bool) (vc.Factory, error) {
```

Comment:
Please add comments for the `fetchOnly` flag here.

Reply:
Will do.